src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
Don't use verbs for class/interface names. Consider renaming the interface to something that represents a noun, such as `AreaCalculator` or `MeasurableArea`, to improve readability and maintain standard naming conventions.